client/http/example_test.go

```diff
@@ -50,3 +48,31 @@ func Example_http_New() {
 
 	//output: got beacon: round=1234; randomness=9ead58abb451d8f521338c43ba5595610642a0c07d0e9babeaae6a98787629de
 }
+
+func Example_http_New_with_chainhash() {
+	var urls = []string{
+		"https://api.drand.sh",
+		"https://drand.cloudflare.com",
+	}
+
+	var chainHash, _ = hex.DecodeString("8990e7a9aaed2ffed73dbd7092123d6f289930540d7651336225dc172e51b2ce")
+
+	ctx := context.Background()
+	lg := log.New(nil, log.DebugLevel, true)
+
+	c, err := client.New(ctx, lg,
+		client.From(http.ForURLs(ctx, lg, urls, chainHash)...),
+		client.WithChainHash(chainHash),
+	)
+	if err != nil {
+		panic(err)
+	}
+
+	info, err := c.Info(context.Background())
+	if err != nil {
+		panic(err)
+	}
+
+	fmt.Println(info.GetSchemeName())
+	//output: pedersen-bls-chained
```

Comment:
noice

Reply:
tis is the `Example` syntax if you want them to actually be run and not just compiled when running `go test`